### DISCUSSION TITLE
How to fetch data- trpc & jotai

### DISCUSSION THREAD

**Discussion by sebaw65** [pmndrs/jotai#2264] [Q&A] [answered]
Hi,

I would like to ask how to retrieve data using trpc query? Analogous to the example without jotai: 
```js
const { data, isLoading } = trpc.anything.findItem.useQuery({
    itemId: id,
  });
```
 

In addition:
- how to get the isLoading parameter (example above)? 
- is it possible to use atoms as hooks as in the example above and pass parameters to them?

Thank you in advance

**Comment by dai-shi** [comment]
https://jotai.org/docs/extensions/trpc ?

**Reply by sebaw65** [reply]
What about returned parameters from `useQuery`, such as `isLoading`? How do you access them?

**Reply by dai-shi** [reply, marked as answer]
We don't have such parameters. https://trpc.io/docs/client/vanilla is just an async function. Thus, jotai-trpc prefers async and Suspense.
